### DISCUSSION TITLE
Question about default audio track

### DISCUSSION THREAD

**Discussion by luckyaigit** [nagadomi/nunif#187] [Q&A]
If the video in mkv format contains more than one audio track, the first track is used by default after conversion, is there any way to specify other audio tracks for output?
![image](https://github.com/user-attachments/assets/c2aaeda3-af19-44b7-a673-85fcc3dbb92f)

**Comment by nagadomi** [comment]
Currently, there is no way to select it.
If way for only case when the input file is a single video file, it can be implemented.

**Reply by luckyaigit** [reply]
Got it. Thanks for the response.

**Comment by diverswan9** [comment]
I would use something like **mkvtoolnix** and add the newly created iw3 file as well as the 'source' video into the top window.  I just use the portable version. Drag and drop video files in top section... tracks will show below... select/un-select the tracks you want. You can add extra audio, subtitles and chapters here if the source video file has them. You can 'Tag' the default audio track as well ( make sure to un-select the video track from the source file ) ... Start Multiplexing (middle button at bottom) [https://mkvtoolnix.download/downloads.html](url)
